sdk/tests/js/webgl-test-utils.js

```diff
@@ -841,11 +841,9 @@ var fillTexture = function(gl, tex, width, height, color, opt_level, opt_format,
   opt_internalFormat = opt_internalFormat || opt_format;
   var pack = gl.getParameter(gl.UNPACK_ALIGNMENT);
   var numComponents = color.length;
-  var bytesPerComponent = getBytesPerComponent(gl, opt_type);
-  var rowSize = numComponents * width * bytesPerComponent;
+  var rowSize = numComponents * width;
   var paddedRowSize = Math.floor((rowSize + pack - 1) / pack) * pack;
```

Comment:
I agree we have a bug here, but this paddedRowSize is wrong if the rowSize isn't in bytes. Some extra attention is needed to get the size right.